src/main/java/duke/tasks/EventTask.java

```diff
@@ -0,0 +1,39 @@
+package duke.tasks;
+
+import duke.commands.Commands;
+import duke.exceptions.InvalidParameterException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class EventTask extends Task {
+
+    private final LocalDate eventTime;
+
+    private EventTask(String taskName, LocalDate eventTime) {
+        super(taskName, "E");
+        this.eventTime = eventTime;
+    }
+
+    public static EventTask EventTaskBuilder(String taskName) throws InvalidParameterException {
```

Comment:
Similar to the comment made in your `DeadlineTask`, it should probably be in the form of a verb (even though this one's quite self-explanatory 🤷‍♂️ 

```suggestion
    public static EventTask buildEventTask(String taskName) throws InvalidParameterException {
```